packages/jest-core/src/runJest.ts

```diff
@@ -159,7 +159,7 @@ export default async function runJest({
   const Sequencer: typeof TestSequencer = await requireOrImportModule(
     globalConfig.testSequencer,
   );
-  const sequencer = new Sequencer();
+  const sequencer = new Sequencer(globalConfig);
```

Comment:
should we pass `contexts` as well, while we're at it?

Reply:
added contexts, although I'm not sure what a possible use case might be, so I haven't added a test.

Any ideas?

Reply:
nah, just doesn't hurt 😅